partner_stage_only_confirmed/models/base.py

```diff
@@ -29,11 +29,16 @@ def get_view(self, view_id=None, view_type="form", **options):
                             if not domain:
                                 domain = "[('state', '=', 'confirmed')]"
                             elif isinstance(domain, str):
-                                if domain in ("", "[]"):
-                                    domain = "[('state', '=', 'confirmed')]"
+                                dom = domain.strip()
+                                if dom in ("", "[]"):
+                                    dom = "[('state', '=', 'confirmed')]"
```

Comment:
```suggestion
                                    domain = "[('state', '=', 'confirmed')]"
```
and we can remove the `domain = dom` line.

Reply:
Done!